### PR TITLE
add remote rename

### PR DESCRIPTION
**Pull request by lonnen**
add the ability to rename a client remotely

### CODE REVIEW

File: static/js/app.js [hunk 1]
```diff
@@ -87,6 +87,14 @@ function untoast() {
   clearTimeout(toastTimeout);
 }
 
+function rename(name) {
+  if (!config.name) {
+    config.name = "";
+  }
```
Comment (mythmon) [inline static/js/app.js]: What's this for?

Comment (lonnen) [inline static/js/app.js]: Defensive code to coerce name to a string if its a number or whatever -or- ritualized cargo cult prelude? You decide.

Comment (mythmon) [inline static/js/app.js]: It won't do that. `!1` is `false`, so it won't trigger this. And anyways you overwrite `config.name` in the next breath, so any decent compiler would optimize this if condition away.

**Comment by mythmon** [comment]
r+ with that silly if condition gone.

**Comment by lonnen** [comment]
:fire: :fire_engine: :fireworks:

**Pull request merged**